orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/WaitForManifestStableTask.java

```diff
@@ -69,6 +69,7 @@ public TaskResult execute(@Nonnull StageExecution stage) {
     List<Map<String, String>> failedManifests = context.getFailedManifests();
     List warnings = context.getWarnings();
     List events = context.getEvents();
+    Boolean includeEvents = context.isIncludeEvents();
```

Comment:
lower-case-b `boolean` is sufficient (and simpler) here ecause the type in WaitForManifestStableContext is lower-case-b `boolean`, right?
```suggestion
    boolean includeEvents = context.isIncludeEvents();
```

Reply:
@dbyron-sf Makes perfect sense. Updated.